mcs/class/System/Test/System.Net.Sockets/TcpClientTest.cs

```diff
@@ -173,7 +173,7 @@ public void ConnectMultiAny ()
 				client.Connect (ipAddresses, 1234);
 				Assert.Fail ("ConnectMultiAny #1");
 			} catch (SocketException ex) {
-				Assert.AreEqual (10049, ex.ErrorCode, "ConnectMultiAny #2");
+				Assert.AreEqual (10061, ex.ErrorCode, "ConnectMultiAny #2");
```

Comment:
This change broke this test on Mono on Windows. 10049 is the error code returned on Windows, both Mono and .NET. Windows 64 test failure is here: https://jenkins.mono-project.com/job/z/label=w64/3796/testReport/junit/MonoTests.System.Net.Sockets/TcpClientTest/ConnectMultiAny/